ci/debian/centreon-broker-storage.install

```diff
@@ -1,2 +1,3 @@
-debian/tmp-centreon-collect/usr/share/centreon/lib/centreon-broker/20-storage.so    usr/share/centreon/lib/centreon-broker
-debian/tmp-centreon-collect/usr/share/centreon/lib/centreon-broker/70-rrd.so        usr/share/centreon/lib/centreon-broker
+debian/tmp-centreon-collect/usr/share/centreon/lib/centreon-broker/20-storage.so        usr/share/centreon/lib/centreon-broker
+debian/tmp-centreon-collect/usr/share/centreon/lib/centreon-broker/20-unified_sql.so    usr/share/centreon/lib/centreon-broker
```

Comment:
Why not add this module into the centreon-broker-core package instead. You are right with the idea that module replaces the storage one but it also replaces the sql one.
So I think its place is better in centreon-broker-core.
Maybe you have another idea.

Reply:
I just followed it as in rpm package. I verified that the file was in the `broker-storage` package:
```
[root@centreon-alma8-1 ~]# rpm -ql centreon-broker-storage
/usr/lib/.build-id
/usr/lib/.build-id/33
/usr/lib/.build-id/33/27c09501c39d36bca00873b32c0ff5f2c4d3de
/usr/lib/.build-id/4c
/usr/lib/.build-id/4c/f1e9f9028118f163281d93e8690339761e7148
/usr/lib/.build-id/d9
/usr/lib/.build-id/d9/adf86e8468f798e32e9c68d7f45bf8c87cab39
/usr/share/centreon/lib/centreon-broker/20-storage.so
/usr/share/centreon/lib/centreon-broker/20-unified_sql.so
/usr/share/centreon/lib/centreon-broker/70-rrd.so
```

Reply:
ok, it's bad :-(